x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/400_connector_sync_job_post.yml

```diff
@@ -0,0 +1,62 @@
+setup:
+  - skip:
+      version: " - 8.11.99"
+      reason: Introduced in 8.12.0
+  - do:
+      connector.put:
+        connector_id: test-connector
+        body:
+          index_name: search-test
+          name: my-connector
+          language: de
+          is_native: false
+          service_type: super-connector
+
+---
+'Create connector sync job':
+  - do:
+      connector_sync_job.post:
+        body:
+          id: test-connector
+          job_type: full
+          trigger_method: on_demand
+  - set:  { id: id }
+  - match: { id: $id }
+
+---
+'Create connector sync job with missing job type':
+  - skip:
+      features: warnings
+  - do:
+      warnings:
+        - Missing field 'job_type'. Using default value 'full'.
```

Comment:
Is it standard practice to return header warnings when we use default values? Would documentation suffice here?

Reply:
I don't know, if it's standard practice, but I've seen it in other places. I thought it could be helpful, so you're aware of values set in the background, which could introduce side effects you don't have in mind. WDYT?

Reply:
Ok you both sold it to me :) Adjusted it with https://github.com/elastic/elasticsearch/pull/102714/commits/46dccc925b277bcbfe8a141c6d704d4e6bf7c8ef.